pkg/kubectl/autoscale.go

```diff
@@ -38,6 +39,7 @@ func (HorizontalPodAutoscalerV1Beta1) ParamNames() []GeneratorParam {
 		{"scaleRef-kind", false},
 		{"scaleRef-name", false},
 		{"scaleRef-apiVersion", false},
+		{"scaleRef-apiGroup", false},
```

Comment:
I agree, but I think the confusing thing is the apiVersion field of TypeMeta, not this field. We should consider changing apiVersion to apiGroupVersion when we rev the api...